logging/logging_alerts/default-logging-alerts.adoc

```diff
@@ -0,0 +1,16 @@
+:_content-type: ASSEMBLY
+[id="default-logging-alerts"]
+include::_attributes/common-attributes.adoc[]
+= Default logging alerts
+:context: default-logging-alerts
+
+toc::[]
+
+Logging alerts are installed as part of the Cluster Logging Operator installation. These alerts depend on the collection of metrics, which is enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the Cluster Logging Operator. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
```

Comment:
nit: Since we have ES Alerts for correctness I've added storage to the source for metrics.
```suggestion
Logging alerts are installed as part of the Cluster Logging Operator installation. These alerts depend on the metrics exported by the collection/storage backends, which are enabled if you selected the option to *Enable operator recommended cluster monitoring on this namespace* when installing the Cluster Logging Operator. For more information about installing logging Operators, see xref:../../logging/cluster-logging-deploying#cluster-logging-deploy-console_cluster-logging-deploying[Installing the {logging-title} using the web console].
```